apps/database/postgres/ormconfig.ts

```diff
@@ -0,0 +1,18 @@
+import { ConnectionOptions } from "typeorm"
+
+const config: ConnectionOptions = {
+  type: "postgres",
+  host: process.env.DATABASE_HOST || "localhost",
+  port: parseInt(process.env.DATABASE_PORT, 10) || 5433,
+  username: process.env.DATABASE_USERNAME || "postgres",
+  password: process.env.DATABASE_PASSWORD || "postgres",
+  database: process.env.DATABASE_NAME || "stator",
+
+  entities: [__dirname + "/../../../**/*.entity{.ts,.js}"],
```

Comment:
use @stator/models here instead of long path

Reply:
It's not supported by the compiler option I need to set for the migrations to work. :(